src/components/carrossel-oficinas/styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+const VerMais = styled.div`
+  .styled-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #f5bc4a;
+    width: 146px;
+    height: 40px;
+    border: none;
+    margin-top: 1rem;
+  }
+  .escritaBotao {
```

Comment:
Eu sugeriria manter a mesma convenção de nomenclatura para todas as regras css, seja camelCase ou kebab-case. Aqui estamos usando as duas. 

Eu particularmente gosto mais do kebab-case então vou sugerir usar esse 😬, mas se preferirem camelCase, acho que é ok também (eu não encontrei nenhuma documentação decente que explique qual é o padrão oficial do CSS).

```suggestion
  .escrita-botao {
```

Reply:
Alteração feita.